src/main/java/com/korailtalk/server/api/timetable/service/TimetableService.java

```diff
@@ -30,15 +30,15 @@ public TimetablesResponse getAllTimetables(Long userId, String date, String depa
     }
 
 
-    @Transactional(readOnly = true)
     private List<TimetableResponse> getTimetableList(Long userId, String date, String departurePlace, String arrivalPlace) {
 
         final User findUser = userRepository.findById(userId).orElseThrow(
                 () -> new NotFoundException(ErrorStatus.NOT_FOUND_USER)
         );
 
-        final List<Timetable> findTimetable = timetableRepository.findByDateAndDeparturePlaceAndArrivalPlace(date, departurePlace, arrivalPlace);
-        return findTimetable.stream().map(
+        final List<Timetable> Timetables = timetableRepository.findByDateAndDeparturePlaceAndArrivalPlace(date, departurePlace, arrivalPlace);
```

Comment:
p1

```suggestion
        final List<Timetable> timetables = timetableRepository.findByDateAndDeparturePlaceAndArrivalPlace(date, departurePlace, arrivalPlace);
```

여기 이 친구 소문자로 바꿔야 할 것 같습니다!